igem_tuebingen_website/translations/de/LC_MESSAGES/messages.po

```diff
@@ -8,7 +8,11 @@ msgid ""
 msgstr ""
 "Project-Id-Version:  1.0.0\n"
 "Report-Msgid-Bugs-To: philipp_ehm@protonmail.com\n"
-"POT-Creation-Date: 2019-06-04 17:55+0200\n"
+<<<<<<< HEAD
```

Comment:
There were also merge conflicts here. Again, tell me if you need help solving them.